react/features/whiteboard/functions.ts

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable import/order */
+import md5 from 'js-md5';
+
+// @ts-ignore
+import { getPinnedParticipant } from '../../features/base/participants';
+import { getRemoteParticipants, isLocalParticipantModerator } from '../base/participants/functions';
+import { appendURLParam } from '../base/util/uri';
+
+// @ts-ignore
+import { getCurrentRoomId, isInBreakoutRoom } from '../breakout-rooms/functions';
+
+// @ts-ignore
+import { electParticipantInLargeVideo } from '../large-video/actions.any';
+import { hideWhiteboard, openWhiteboard, showWhiteboard } from './actions';
+import { WHITEBOARD_ID } from './constants';
+
+const getWhiteboardState = (state: any): any => state['features/whiteboard'];
+
+// TODO: remove negation
```

Comment:
Why the negation?

Reply:
Just for me to easily deploy and test without config values, I'll remove it